service_worker.js

```diff
@@ -1,119 +1,24 @@
+import { createRules } from './rules.js';
+
 chrome.storage.sync
   .get({
-    match: "https://my-source.com",
-    replace: "http://localhost:8080",
+    match: 'https://my-source.com',
+    replace: 'http://localhost:8080',
     wds: true,
     webServerPort: 8080,
-    compressed: false
+    compressed: false,
   })
-  .then(async ({ match, replace, wds, webServerPort, compressed }) => {
-    const replaceURL = new URL(replace);
-    const likelyWebServerURL = `${replaceURL.protocol}//${replaceURL.hostname}:${webServerPort}`;
-
-    const isWDS = wds === "true";
-    const isCompressed = compressed === "true";
-    const matchForRegex = match.replaceAll(/\./g, '\\.');
-    const likelyWebServerURLForRegex = likelyWebServerURL.replaceAll(/\./g, '\\.');
-
-    const genRegex = `^${matchForRegex}.*\/static\/([a-z]+)\/([^/]*)\\.[a-fA-F0-9]+\\..*$`;
-    const appAssetRegex = `^${matchForRegex}.*\/static\/([a-z]+)\/([a-z]+)\/(.*)\\.[a-fA-F0-9]+\\..*$`
-    const appAssetNoHashRegex = `^${matchForRegex}.*\/static\/([a-z]+)\/([a-z]+)\/(.*)\\..*$`
+  .then(async (config) => {
+    // Uncomment for seeing matches (not a lot of info).
+    chrome.declarativeNetRequest.onRuleMatchedDebug.addListener((info) => {
+      console.log(JSON.stringify(info));
+    });
```

Comment:
Looks like we'd had this commented out before, should we leave that as the better default? Or if not, we should at least update the comment.